runtime/lua/editorconfig.lua

```diff
@@ -1,10 +1,9 @@
 --- @brief
---- Nvim supports EditorConfig. When a file is opened, Nvim searches all parent
---- directories of that file for ".editorconfig" files, parses them, and applies
---- any properties that match the opened file. Think of it like 'modeline' for an
---- entire (recursive) directory. For more information see
---- https://editorconfig.org/.
----
+--- Nvim supports EditorConfig. When a file is opened, after running |ftplugin|s
```

Comment:
Is this too wordy?

```suggestion
--- Nvim supports EditorConfig. When a file is opened, after running |ftplugin|s (including the ones in `after/`)
```

Reply:
I don't think we need this, `after/ftplugin` is still part of "ftplugins".